packages/taquito-local-forging/src/protocols.ts

```diff
@@ -25,7 +26,8 @@ const protoLevel: Record<ProtocolsHash, number> = {
   PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx: 11,
   Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A: 12,
   PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY: 13,
-  ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK: 14,
+  PtKathmanYPEsBau83zRwXK19HAQRzZSj6pCXnvtyLZFjRxdQxt: 14,
```

Comment:
It should be PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg.

Reply:
fixed